docs/attributes.md

```diff
@@ -0,0 +1,47 @@
+# Object Attributes (attrs)
+
+# This is still a work in progress (mistakes are common).
+
+## For objects that have associated events
+> Required in **bold**
+- duration: number; The number of users that can be admitted in the event associated to the event.
```

Comment:
mmmm it seems the description of `capacity`

Reply:
I think capacity and eventDuration (and maybe registrationDuration, at least for now) should be required. We have default values in casus nothing is set (1 or 0) but then it's useless

Reply:
Yes. I'm actually going to remove this file. Because it's not ready

Reply:
I thought about it more and maybe it's good to have it in the mean time before we organize and setup the automatic documentation generated from the code.

Reply:
okok - I see a new file `object-attributee-reference`, do you want me to read it then? or you want to rm it from the doc?